src/NzbDrone.Core/Parser/Model/FindMovieResult.cs

```diff
@@ -0,0 +1,24 @@
+using NzbDrone.Core.Movies;
+
+namespace NzbDrone.Core.Parser.Model
+{
+    public class FindMovieResult
+    {
+        public Movie Movie { get; set; }
+        public MovieMatchType MatchType { get; set; }
+
+        public FindMovieResult(Movie movie, MovieMatchType matchType)
+        {
+            Movie = movie;
+            MatchType = matchType;
+        }
+    }
+
+    public enum MovieMatchType
+    {
+        Unknown = 0,
+        Title = 1,
+        Alias = 2,
+        Id = 3
```

Comment:
Could/should we go even more granular?

```suggestion
        Unknown = 0,
        TitleReleaseYear = 1,
        TitlePremierYear = 2,
        AltTitle = 3,
        TranslateTitle = 4,
        IdTMDb = 5,
        IdIMDbId = 6
```

Reply:
I'll add the translation/alt title instead of alias, I don't think we need to break out ids though, it gets crazy